src/user-interface/selectors.js

```diff
@@ -0,0 +1,13 @@
+import { createSelector } from 'reselect';
+
+
+export function getMenuIsOpenState(state) {
+  return state.userInterface.isMenuOpen;
```

Comment:
Same here and the next function can be arrow functions without the `return` statement.